doc/README/en/README.md

```diff
@@ -145,88 +143,83 @@ Crowdin, the markdown files could contain unnescessary lines, which lead to brok
 not needed code sections for example.
 
 ### Validate JSDoc tool.
-The tool `validatejsdoc` allow verification of the current implementation of JSDoc with reference implementation.
-It was used during porting JSDoc to the Node version of JSDoc, and now currently not used in the workflow.
+
+The tool `validatejsdoc` allow verification of the current implementation of JSDoc with reference implementation. It was used during porting JSDoc to the Node version of JSDoc, and now currently not used in the workflow.
 
 Recommendations for the translators
--------------------------
-If you intend to create quality translation of the Cordova docs, please not only
-work in Crowdin and translate documentation, but also please go extra mile and verify that
-generated documentation for your language is also produce quality results.
+-----------------------------------
+
+If you intend to create quality translation of the Cordova docs, please not only work in Crowdin and translate documentation, but also please go extra mile and verify that generated documentation for your language is also produce quality results.
```

Comment:
Grammar Fixes:
If you intend to create a quality translation of the Cordova docs, please don't only translate the documentation in Crowdin, but also go the extra mile and verify that the generated documentation for your language is producing quality results.